language/move-compiler/tests/move_check/typing/declare_duplicate_binding.exp

```diff
@@ -26,6 +26,12 @@ error[E02001]: duplicate declaration, item, or annotation
   │              │     
   │              Previously declared here
 
+warning[W09002]: unused variable
+  ┌─ tests/move_check/typing/declare_duplicate_binding.move:6:19
+  │
+6 │         let (f, R{f}, f);
```

Comment:
It's because I assign to `f`. Is this a confusing error? Not sure how common it will be.

With the numbers expanded it's something like
```
let (f#1, R { f: f#2 }, f#3);
f#3 = 0; f#3;
```